maintainers/maintainer-list.nix

```diff
@@ -12024,4 +12024,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  simonkampe = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
It already is a separate commit (3839fc0), should I put it in a separate PR? I will fix the sorting.

Reply:
No, just fix the sorting.

Reply:
Sorting is fixed.